src/Middleware/RequestThrottling/ref/Microsoft.AspNetCore.RequestThrottling.netcoreapp3.0.cs

```diff
@@ -30,6 +24,11 @@ public partial class RequestThrottlingOptions
 }
 namespace Microsoft.AspNetCore.RequestThrottling.QueuePolicies
 {
+    public partial interface IQueuePolicy
```

Comment:
I don't mind organizing the directory structure this way, but I think all the public types should live in the Microsoft.AspNetCore.RequestThrottling namespace. Having to "use" two namespaces just to use this feature would be pretty annoying.

Reply:
As a general rule do not create a new sub-namespace if it's only going to have a few types in it. (< 5?)